File: Bouquet.Api/Bouquet.Database/Entities/ShopConfig.cs
namespace Bouquet.Database.Entities
{
    public class ShopConfig
    {
        public string Id { get; set; }

        public TimeSpan OpenAt { get; set; }

        public TimeSpan CloseAt { get; set; }

        public TimeSpan SameDayTillHour { get; set; }

        public double Price { get; set; }

        public double FreeDeliveryAt { get; set; }

        public string FlowerShopID { get; set; }

        #region Navigation database properties

        /// <summary>
        /// Магазин
        /// </summary>
        public FlowerShop? FlowerShop { get; set; }

        #endregion

        public ShopConfig()
        {
            Id = Guid.NewGuid().ToString();
        }
    }
}

File: Web.Client/src/resources/types/shop/ShopConfig.ts
import TimeSpan from "@web-atoms/date-time/src/TimeSpan"

export default interface ShopConfig {
    id: string;
    price: number;
    freeDeliveryAt: number;
    openAt: TimeSpan;
    closeAt: TimeSpan;
    sameDayTillHour: TimeSpan;
}